code/modules/martial_arts/bearserk.dm

```diff
@@ -38,8 +38,10 @@
 
 // The Pelt
 
-/obj/item/clothing/head/bearpelt/bearserk
-	armor = list(MELEE = 30, BULLET = 20, LASER = 20, ENERGY = 20, BOMB = 20, RAD = 0, FIRE = 10, ACID = 10)
+/obj/item/clothing/head/bearpelt/
```

Comment:
the path is incomplete?

Reply:
That would explain some issues. I wonder how long this has been like this... doesn't matter, thanks for noticing!